packages/workbox-core/models/messages/messages.mjs

```diff
@@ -23,4 +23,33 @@ export default {
     }
     return `The '${moduleName}' must be used in a service worker.`;
   },
+
+  'not-an-array': ({moduleName, className, funcName, paramName}) => {
+    if (!moduleName || !className || !funcName || !paramName) {
+      throw new Error(`Unexpected input to 'not-an-array' error.`);
+    }
+    return `The parameter '${paramName}' passed into ` +
+      `'${moduleName}.${className}.${funcName}()' must be an array.`;
+  },
+
+  'add-to-cache-list-unexpected-type': ({entry}) => {
+    return `An unexpected entry was passed to ` +
+    `'workbox-precaching.PrecacheManager.addToCacheList()'. The entry ` +
```

Comment:
Is there period supposed to be there at the end of ".addToCacheList()'."? Same question for line 50.